src/omero/plugins/duplicate.py

```diff
@@ -51,7 +46,23 @@
     # that would have been duplicated
     omero duplicate Dataset:53 --dry-run --report
 
+    # Duplicate a project with its datasets but not their images
+    omero duplicate Project:15 --ignore-classes=DatasetImageLink
+    # Duplicate a project with the original images linked from its datasets
+    omero duplicate Project:15 --reference-classes=Image
 """
+        "    # Duplicate a project, linking to the original annotations "
+        "except for duplicating the comments and ratings\n"
+        "    omero duplicate Project:15 --reference-classes=Annotation "
+        "--duplicate-classes=CommentAnnotation,LongAnnotation\n"
+        """
+Group permissions can prevent simply referencing an Image or Annotation.
```

Comment:
```suggestion
Group permissions can prevent simply linking an Image or Annotation.
```

Reply:
Maybe this is not optimal suggestion.
How would "referencing for linking" sound ? Or maybe ``Group permissions can prevent simply referencing an Image or Annotation because the subsequent linking is not possible.``

?